### PR TITLE
ZIOS-10195: Remove CocoaPods from build scripts

### PR DESCRIPTION
**Pull request by alexaubry**
### Issues

We decided to stop using CocoaPods as a dependency manager. This pull requests removes all traces of CocoaPods from the build scripts.

### CODE REVIEW

File: environment.sh [hunk 1]
```diff
@@ -12,7 +12,7 @@ if [ "$CIRCLE_PROJECT_REPONAME" == "wire-ios" ];
 then
 	echo "export IS_UI_PROJECT=1" >> $BASH_ENV
 	echo "export SCHEME='Wire-iOS'" >> $BASH_ENV
-	echo "export WORKSPACE='Wire-iOS.xcworkspace'" >> $BASH_ENV
+	echo "export PROJECT='Wire-iOS.xcodeproj'" >> $BASH_ENV
```
Comment (vytis) [inline environment.sh]: We need to check if we use this variable anywhere actually

File: test.sh [hunk 2]
```diff
@@ -10,7 +10,7 @@ arguments+=( -derivedDataPath DerivedData )
 
 if [ "$IS_UI_PROJECT" -eq "1" ]; 
 then
-	arguments+=( -workspace "${WORKSPACE}" ) # We need to append workspace argument for UI project
+	arguments+=( -project "${PROJECT}" ) # We need to append workspace argument for UI project
```
Comment (vytis) [inline test.sh]: This should not be needed I think, frameworks build fine without this argument

**Comment by daehn** [comment]
@alexaubry also please remove `CocoaPods` from the `Gemfile` itself, as the `Gemfile.lock` is generated when bundler is run (basically like `Cartfile.resolved` and `Podfile.lock`) and specifies the actual versions being used.

**Comment by alexaubry** [comment]
@daehn I removed CocoaPods from `setup.sh` and the `Gemfile` in a PR in UI (see [#2368](https://github.com/wireapp/wire-ios/pull/2368)).

**Comment by daehn** [comment]
@alexaubry yeah I mixed it up and mistook this PR to be the UI PR.

**Pull request merged**